web-client/src/presenter/actions/getDocumentContentsAction.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * gets documentContents based on props.docketEntryIdToEdit
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.applicationContext the cerebral props object
+ * @param {object} providers.props the cerebral props object
+ * @returns {object} the document contents
+ */
+export const getDocumentContentsAction = async ({
+  applicationContext,
+  props,
+}) => {
+  const { caseDetail, docketEntryIdToEdit } = props;
+
+  if (docketEntryIdToEdit) {
+    const { documentContentsId } = caseDetail.docketEntries.find(
+      d => d.docketEntryId === docketEntryIdToEdit,
+    );
+
```

Comment:
Should this action throw an error if 
a) it doesn't receive a docketEntryIdToEdit
b) it can't find docketEntryIdToEdit in caseDetail.docketEntries

Reply:
That flow _shouldn't_ get hit; furthermore, there are other actions that do this and don't throw an error for the same reason. Here's a few, for example:

- completeWorkItemForDocumentSigningAction.js
- getDocumentEditUrlAsPathAction.js
- setDocumentToEditAction.js

Reply:
I agree that logically shouldn't happen. However, if it does, it will fail pretty quietly. No need to fix now, but it's something to consider for future actions that we write.